docs/specs/skills-system/skills-syncing-kiro.md

```
@@ -0,0 +1,283 @@
+
+| 属性 | 值 |
+|------|-----|
+| 工具名称 | Kiro |
+| Steering 目录 | `.kiro/steering/` |
+| 文件模式 | `*.md` |
+| 格式类型 | YAML frontmatter + Markdown |
+| Frontmatter | **可选** |
+
+## 2. 目录结构
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)